src/views/open-api-docs-view-v2/components/operation-content/server.ts

```diff
@@ -4,31 +4,30 @@
  */
 
 // Utils
-import { getOperationObjects } from '../../utils/get-operation-objects'
+
 import { getUrlPathCodeHtml } from '../../utils/get-url-path-code-html'
 import { truncateHcpOperationPath } from '../../utils/truncate-hcp-operation-path'
 import { getRequestData } from '../../utils/get-request-data'
 import { getResponseData } from '../../utils/get-response-data'
+import { slugifyOperationId } from 'views/open-api-docs-view-v2/utils/slugify-operation-id'
 // Types
 import type { OpenAPIV3 } from 'openapi-types'
 import type { OperationContentProps } from '.'
+import type { OperationObject } from '../../utils/get-operation-objects'
 
 /**
  * Transform the schemaData into props for an individual operation
  */
 export default async function getOperationContentProps(
```

Comment:
Made some minor modifications here to avoid passing duplicate data, and to lift the `targetOperation` determination process into the enclosing `generateStaticProps`.